doc/api/next_api_changes/behavior/20405-JMK.rst

```diff
@@ -0,0 +1,8 @@
+Make norm from scale now public
+===============================
+
+Downstream libraries can take advantage of ``make_norm_from_scale`` in 
+``colors.py`` to create a `~.colors.Normalize` subclass directly from 
```

Comment:
Probably should just make this a full-on reference?

Reply:
Yeah, for some reason that wasn't working...

Reply:
Ah, an entry needs to be added to `doc/api/colors_api.rst`.

Reply:
I think this still needs to be changed to e.g.
```rst
take advantage of `.colors.make_norm_from_scale`
```
?